contracts/Manager.sol

```diff
@@ -795,6 +1029,17 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
         return false;
     }
 
+    /**
+     * @notice Returns which strategy is account using
+     * address(0) = default strategy
+     * !address(0) = voting for specific validator group
```

Comment:
Did we move away from the design that non-0 addresses that are not allowed groups also mean to use the default strategy for now, until the group becomes allowed? Not sure if that's going to work.

Reply:
Updated - currently it is possible to vote only for allowed strategies